waterlog/src/Components/BtnResolve.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchSegmentsLeaksResolve } from '../actions/SegmentResolveActions';
+import { Globals } from './../Globals';
+
+class BtnResolve extends Component {
```

Comment:
Just fix your indentation

Reply:
Done

Reply:
Done please review again

Reply:
Yes I'm sure

Reply:
@MsweliP What Kimi is pointing out. Your project has used tab 2 in other files [eg:](https://github.com/Hunter27/WaterLog-Front-end/blob/master/waterlog/src/Components/Tank.js) and now here you are using tab 4